packages/webapp/app/providers.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import type { PropsWithChildren, ReactElement } from 'react';
+import React from 'react';
+import { QueryClientProvider } from '@tanstack/react-query';
+import { getQueryClient } from '@dailydotdev/shared/src/graphql/queryClient';
+import { queryClientAtom } from 'jotai-tanstack-query';
+import { useHydrateAtoms } from 'jotai/react/utils';
+import { Provider as JotaiProvider } from 'jotai/react';
+
+const HydrateAtoms = ({ children }) => {
+  const queryClient = getQueryClient();
+  useHydrateAtoms([[queryClientAtom, queryClient]]);
+  return children;
+};
+
+export default function Providers({
+  children,
+}: PropsWithChildren): ReactElement {
+  const queryClient = getQueryClient();
+
+  return (
+    <QueryClientProvider client={queryClient}>
+      <JotaiProvider>
```

Comment:
With the stuff we talked about, do we want to use Jotai since it provides a lot of limitations and author itself recommends not using it? Does Jotai help us with anything specific for this feature?

Reply:
I was imagining to use for the stepper logic, maybe if not needed I can remove. Wdyt?

Reply:
Yeah not sure what is entitled in stepper logic, just currently I don't see we use it for anything we can't archive with state or useReducer.

Reply:
Removed for now ✔️